### ISSUE TITLE
Twilightboxart isn't working

### ISSUE THREAD

**Issue by echabon** [KirovAir/TwilightBoxart#19]
I keep getting this message: "An error occurred while sending the request. The remote name could not be resolved: 'boxart.kirovair.com'"

I recently modded my girlfriends dsi xl and I like the look of box arts she doesn't really mind it, but it irritates me not seeing box art. thanks.

**Comment by ffrancoa** [comment]
I think duplicated from https://github.com/KirovAir/TwilightBoxart/issues/17. Sometimes the error message varies, but basically it is what the developer mentions: "`the sponsored server has been shut down which handled the resizing for the DS clients`".

**Comment by Masamune3210** [comment]
Indeed. For now, DS side box art downloads are down, and will likely be down for quite a while until if/when a new system can be put in place.

**Comment by linktoalec** [comment]
hey you guys, download the version 0.6 of TwilightBoxart, is working a little bit slow but working, i will leave a screenshot of today to prove is working 
![image](https://github.com/KirovAir/TwilightBoxart/assets/146513712/10040b76-33cd-4aa8-baf9-534ce795f60e)

**Comment by e0406370** [comment]
> hey you guys, download the version 0.6 of TwilightBoxart, is working a little bit slow but working, i will leave a screenshot of today to prove is working ![image](https://user-images.githubusercontent.com/146513712/271757151-10040b76-33cd-4aa8-baf9-534ce795f60e.png)

Can confirm the version 0.6 works!

**Comment by xKarysxd** [comment]
YUP, 0.6 works fine! 
![image](https://github.com/KirovAir/TwilightBoxart/assets/120349025/e92a0f71-c3a7-4d74-b2c8-669406efb0a6)

**Comment by soldiaz** [comment]
confirm but just finds 2 snes covers

**Comment by KirovAir** [comment]
Sorry for the wait, the server was offline for a long while. TwilightBoxart 2.0 is out and everything works again. Fun one for you: there's now a DSi homebrew client, so the console can download its own box art over WiFi without touching the SD card. https://github.com/KirovAir/TwilightBoxart/releases/tag/2.0.0
